docs/app/pages/components/sections/date-time-picker/time-picker/snippets/app.html

```diff
@@ -0,0 +1,86 @@
+<timepicker [ngModel]="myTime" [min]="minTime" [max]="maxTime" [hourStep]="hstep" [minuteStep]="mstep" [showMeridian]="ismeridian"
+    [showSeconds]="showSec" [readonlyInput]="!isEnabled" [mousewheel]="mousewheel" [arrowkeys]="arrowkeys" (ngModelChange)="changed($event)">
+</timepicker>
+
+<p class="m-t">Time is: <em>{{myTime}}</em></p>
+
+<div class="row uxd-customize-example">
+    <div class="col-md-12">
+        <accordion>
+            <accordion-group heading="Customize Example...">
+                <div class="row uxd-customize-row">
+                    <div class="col-md-6 col-sm-6">
+                        <div>
+                            <label for="minHour">min hour</label>
```

Comment:
Isn't the output of a timepicker a Date object? Which is the same as the max and min properties expect. I don't think string manipulation is required.